src/rotest/api/resource_control/cleanup_user.py

```diff
@@ -0,0 +1,46 @@
+# pylint: disable=unused-argument, no-self-use
+import httplib
+
+from django.db import transaction
+from swaggapi.api.builder.server.response import Response
+from swaggapi.api.builder.server.request import DjangoRequestView
+
+from rotest.management import ResourceData
+from rotest.management.common.utils import get_username
+from rotest.api.common.responses import CleanupUserResponseModel
+
+
+class CleanupUser(DjangoRequestView):
+    """Cleaning up user's requests and locked resources."""
+    URI = "resources/cleanup_user"
+    DEFAULT_RESPONSES = {
+        httplib.OK: CleanupUserResponseModel,
+    }
+    TAGS = {
+        "post": ["Resources"]
+    }
+
+    def post(self, request, *args, **kwargs):
+        """Cleaning up user's requests and locked resources.
+
+        Args:
+            username (str): the username to cleanup.
+
+        Returns:
+            SuccessReply. a reply indicating on a successful operation.
+        """
+        user_name = get_username(request)
+        with transaction.atomic():
+            resources = ResourceData.objects.select_for_update().filter(
+                owner=user_name)
+
+            if resources.count() == 0:
+                return Response({
+                    "details": "User {} didn't lock any resource".format(
```

Comment:
Why you're treating those cases differently?
The case where the user doesn't use any resource, is just a case where he locks 0 resources...

Reply:
This is copy pasted from the original code.
I am not sure what u  meant here.
U want me to return "User {} was successfully cleaned always"?

Reply:
I want lines 37-41 deleted